content/docs/rfcs/72/README.md

```diff
@@ -202,11 +205,14 @@ Hashing function used: Poseidon Hash, as described in [Poseidon Paper](https://e
 
 `version`: "0.2" 
 
-`hash_function`: "poseidonHash"
+`hashFunction`: "poseidonHash"
```

Comment:
does this hashFunction refer to the hash function used by the rln protocol, or by the keystore? it is necessary to differentiate between the two because we use sha256 to generate the keys for the multiple credentials in the keystore, and poseidon in the protocol which will not change anytime soon. if we do decide to move from poseidon to a different hash function, that will be reflected in the keystore `version` field, which we will update and ensure the rfc has the reference to it for other implementers as well.

Reply:
@rymnc clarified where poseidon is being used in the test vector section. But in the `memebersipHash` creation  SHOULD is mentioned for SHA256 and other MAY be used.  Should I clarify here as well?